docs/UserManuals/ConfigUI/GitHub.md

```diff
@@ -120,6 +120,13 @@ If you're using GitHub Action to conduct `deployments`, please select "Detect De
 - Deployment: A GitHub Action job with a name that matches the given regEx will be considered as a deployment.
 - Production: A GitHub Action job with a name that matches the given regEx will be considered a job in the production environment.
 
+A GitHub workflow run has many jobs. Each GitHub workflow run is converted to a 
+cicd_pipeline in the domain layer and each GitHub Action job is converted to a cicd_task in the domain layer.
+![github-action-run](/img/ConfigUI/github-action-run.png)
+![github-action-job](/img/ConfigUI/github-action-job.png)
+
+The deployment and production regex is always applied to the records in the cicd_tasks table.
+
 By the above two fields, DevLake can identify a production deployment among massive CI jobs.
```

Comment:
"The regexes allow users to filter their CI/CD pipelines for deployments (to production)."

Reply:
fixed

Reply:
delete "By the above two fields, DevLake can identify a production deployment among massive CI jobs."